### ISSUE TITLE
`Style/OperatorMethodCall` false positive

### ISSUE THREAD

**Issue by aglushkov** [rubocop/rubocop#11508]
C: [Correctable] Style/OperatorMethodCall: Redundant dot detected.

```ruby
  TimeCalc.now.-(9, :minutes) # https://github.com/zverok/time_calc
```

After auto-correct ruby syntax becomes invalid

## Expected behavior

It should not warn if operator used with multiple parameters.

## RuboCop version
```
1.44.1 (using Parser 3.2.0.0, rubocop-ast 1.24.1, running on ruby 3.2.0) [x86_64-linux]
  - rubocop-rspec 2.18.1
 
```
